src/cascadia/TerminalSettingsModel/FileUtils.cpp

```diff
@@ -123,7 +123,15 @@ namespace Microsoft::Terminal::Settings::Model
 
     void WriteUTF8FileAtomic(const std::filesystem::path& path, const std::string_view content)

```

Comment:
nit: You could modify this function to take `path` as a by-value argument.
That way you don't have to create the local `resolvedPath` copy.

Reply:
Thanks for the review!

This could simplify the code for sure. On the other hand it would probably make the interface with the clients less clean: i.e., tomorrow we might decide to implement the function completely differently, and then we will need to revert the interface to the current signature.

Reply:
You don't have to change it. 🙂

But in general: Since we aren't writing a library, we can still change almost any code `src/cascadia` however we want to. This file in particular is an internal implementation detail.

Reply:
> But in general: Since we aren't writing a library, we can still change almost any code `src/cascadia` however we want to. This file in particular is an internal implementation detail.

Yeah I know - and in general you are right! My point is that every class (especially like this util) might become a library some day, so I usually prefer not to pass by value even in such cases. But this more of a habit I guess :blush: